share/gpodder/extensions/concatenate_videos.py

```diff
@@ -38,8 +38,8 @@ def _get_save_filename(self):
         dlg = Gtk.FileChooserDialog(title=_('Save video'),
                 parent=self.gpodder.get_dialog_parent(),
                 action=Gtk.FileChooserAction.SAVE)
-        dlg.add_button(Gtk.STOCK_CANCEL, Gtk.ResponseType.CANCEL)
-        dlg.add_button(Gtk.STOCK_SAVE, Gtk.ResponseType.OK)
+        dlg.add_button('_Cancel', Gtk.ResponseType.CANCEL)
```

Comment:
aren't you breaking localization of the buttons? Or is '_Cancel' automatically translated by gtk?

Reply:
I don't think it is translated, see https://stackoverflow.com/questions/36805505/gtk-stock-is-deprecated-whats-the-alternative/36811163#36811163.

I haven't had time to look at this PR yet, but we should also fix the menu items that currently use _('_Cancel') and _('_Open'), if not already done by this PR.

Reply:
Yeah, `_()` has to be used for gettext (`_ = gpodder.gettext` is there in the global scope for this).

Reply:
Well, according to that link, we should be used dgettext. Declaring `D_ = t.dgettext` and then using `D_('gtk3', '_Cancel')` to translate strings known to gtk.

Reply:
The localization breaks, although it should not according to docs. Using `D_('gtk30', '_Blah')` for all the GTK strings works, but is a bit of an eyesore.

Here's a patch on top of this PR which makes the translation of the Close button in the About dialog work again. Should we use this style, or maybe we can just replace all the GTK strings with local translations, like `_('Blah')`?

```--- a/src/gpodder/__init__.py
+++ b/src/gpodder/__init__.py
@@ -129,6 +129,8 @@ if ui.win32:
     from gpodder.utilwin32locale import install
     install(textdomain, locale_dir)
 
+dgettext = gettext.dgettext
+
 t = gettext.translation(textdomain, locale_dir, fallback=True)
 
 gettext = t.gettext
diff --git a/src/gpodder/gtkui/app.py b/src/gpodder/gtkui/app.py
index b77baa05..0d8913d4 100644
--- a/src/gpodder/gtkui/app.py
+++ b/src/gpodder/gtkui/app.py
@@ -45,6 +45,7 @@ logger = logging.getLogger(__name__)
 
 _ = gpodder.gettext
 N_ = gpodder.ngettext
+D_ = gpodder.dgettext
 
 
 def parse_app_menu_for_accels(filename):
@@ -202,7 +203,7 @@ class gPodderApplication(Gtk.Application):
     def on_about(self, action, param):
         dlg = Gtk.Dialog(_('About gPodder'), self.window.gPodder,
                 Gtk.DialogFlags.MODAL)
-        dlg.add_button('_Close', Gtk.ResponseType.OK).show()
+        dlg.add_button(D_('gtk30', '_Close'), Gtk.ResponseType.OK).show()
         dlg.set_resizable(True)
 
         bg = Gtk.Box(orientation=Gtk.Orientation.VERTICAL, spacing=6, margin=16)
```

Reply:
_Cancel and _Open already use _() and are untranslated for most languages, but using D_ will give them translations for all languages, no matter how verbose they will become.

Reply:
We could define `Dgtk_(s) = lambda: s D_('gtk30', s)` and use `Dgtk_('_Close')` for GTK strings. Would this be ok?

Reply:
The Gtk.STOCK label replacement strings are now translated with GTK translations.